.github/workflows/prod-build.yml

```diff
@@ -432,10 +460,11 @@ jobs:
 
       - name: Update AI Help index with macros
         working-directory: mdn/yari
-        run: yarn ai-help-macros update-index
+        run: yarn ai-help-macros update-index "$FRED_ROOT/out/en-us/docs"
         env:
           OPENAI_KEY: ${{ secrets.OPENAI_KEY }}
           PG_URI: ${{ secrets.PG_URI }}
+          FRED_ROOT: ${{ github.workspace }}/mdn/fred
```

Comment:
Nit: (dito, should really be `$BUILD_OUT_ROOT`, at job-level, since it's used so much)